### PR TITLE
Features/#333 spatial clustering for sector coupling

### PR DESCRIPTION
**Pull request by CarlosEpia**
The applied changes allow to cluster all the different sectors around the eHV grid.

### CODE REVIEW

**Comment by ClaraBuettner** [comment]
I finally found the problem with the empty `etrago.network.loads_t.p_set`; I set `with_time ` to `False `here:
https://github.com/openego/eTraGo/blob/630a2189a2003eab4c2e02fb17d4331df6a76ebd/etrago/cluster/networkclustering.py#L424-L427
I think I did this  because there were some problems when no load data was available. But now there is load data available, and to consider it in clustering this parameter has to be set to `True` again. Sorry for the confusion! I fixed this in my latest commit.

File: etrago/appl.py [hunk 1]
```diff
@@ -327,8 +327,8 @@ def run_etrago(args, json_path):
         eTraGo result network based on `PyPSA network
         <https://www.pypsa.org/doc/components.html#network>`_
     """
-    etrago = Etrago(args, json_path)
-
+    etrago = Etrago(args, json_path= None)
```
Comment (ClaraBuettner) [inline etrago/appl.py:330]: Do you want to merge this? It looks more like a temporal adjustment...

File: etrago/cluster/networkclustering.py [hunk 2]
```diff
@@ -125,8 +217,8 @@ def cluster_on_extra_high_voltage(network, busmap, with_time=True):
     network.generators.control = "PV"
     network.generators['weight'] = 1
 
-    network.generators = network.generators[~network.generators.carrier.isin(
-        ['gas', 'solar_thermal_collector', 'geo_thermal'])]
+    #network.generators = network.generators[~network.generators.carrier.isin(
+    #    ['gas', 'solar_thermal_collector', 'geo_thermal'])]
```
Comment (ClaraBuettner) [inline etrago/cluster/networkclustering.py:221]: These lines could be deleted

File: etrago/appl.py (hunk 1 above)
Comment (CarlosEpia) [reply etrago/appl.py]: The "= None" was deleted.

File: etrago/cluster/networkclustering.py (hunk 2 above)
Comment (CarlosEpia) [reply etrago/cluster/networkclustering.py]: The mentioned lines were deleted.

**Pull request merged**